CONTRIBUTING.md

```diff
@@ -54,6 +54,20 @@ see [ABI Policy Guide](sycl/doc/ABIPolicyGuide.md) for more information.
 - Run regression tests -
 [instructions](sycl/doc/GetStartedGuide.md#test-dpc-toolchain).
 
+**Guidelines for adding Clang FE tests**:
```

Comment:
NOTE: these are guidelines for **SYCL** Clang FE tests only.

I suppose similar guide lines exist for SYCL library project. Should we put them in root CONTRIBUTING.md file or better have a project specific file referenced from the root CONTRIBUTING.md file?

Reply:
There is NB note above with reference to [Test DPC++ toolchain](sycl/doc/GetStartedGuide.md#test-dpc-toolchain).
I think text there and these guidelines added should be combined so that all guidelines on tests development can be found in single place.
It probably can be done at this high-level CONTRIBUTING.md for now as Test Development section and we can decide to move it elsewhere later.

Reply:
So we want the [Test DPC++ toolchain](https://github.com/intel/llvm/blob/174b535596c337ba2cffd3da34e4bde1b7b66da1/sycl/doc/GetStartedGuide.md#test-dpc-toolchain) section moved here and I could add a new sub-section in this doc to describe SYCL Clang FE test guidelines.
Is that correct?

Reply:
Correct, but I think we should move here only first part of "Test DPC++ toolchain" section, i.e. text between "Test DPC++ toolchain" and "Run in-tree LIT tests" headers, basically the part which explains where tests are and where changes to tests should be contributed.
It seems FE guidelines belong to "DPC++ in-tree LIT" part of that, so you can probably try to integrate FE guidelines over there or have some separate sub-section for FE nearby.

Reply:
Got it, Thanks! Do we want to remove the contents between `"Test DPC++ toolchain"` and `"Run in-tree LIT tests"` headers from the GSG? I wasn't sure about that, so I haven't removed that text from GSG.

Reply:
Yes, please. We should avoid information duplication, it's going to be harder to maintain.